### PR TITLE
Add project/id/members endpoint

### PR DESCRIPTION
**Pull request by emintham**
- Removes users/groups from projects endpoint.
- Adds users/groups to projects/id/members endpoint.

Refs #SDE-3324

### CODE REVIEW

**Comment by hanifvirani** [comment]
This will a v4.4 change right?

File: source/includes/project_members.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+    }],
+    "groups": [{
+        "id": "G1",
+        "name": "Devs",
+        "role": "PR4"
+    }]
+}
+```
+
+This endpoint returns the membership information for the specified project.
```
Comment (hanifvirani) [inline source/includes/project_members.md]: Maybe we should mention here that by default we are only returning direct users and groups.

File: source/includes/project_members.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+        "is_active": true
+    }, {
+        "id": "4",
+        "email": "eve@example.com",
+        "role": "PR4",
+        "first_name": "Eve",
+        "last_name": "Online",
+        "is_active": true
+    }],
+    "groups": [{
```
Comment (emintham) [inline source/includes/project_members.md]: Yes

File: source/includes/project_members.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+Content-Type: application/json
+
+{
+    "users": [{
+        "id": "1",
+        "email": "test@example.com",
+        "role": "PR4",
+        "first_name": "Admin",
+        "last_name": "Testerton",
+        "is_active": true
```
Comment (hanifvirani) [inline source/includes/project_members.md]: ```is_active``` is at a global level and not a project level right? I think this field would be read only and not something we expect when we PUT/PATCH.

**Comment by hanifvirani** [comment]
One more comment, but +1 otherwise.

File: source/includes/project_members.md (hunk 3 above)
Comment (emintham) [inline source/includes/project_members.md]: Fixed

**Pull request merged**